src/Administration/Resources/app/administration/src/app/component/form/sw-price-field/sw-price-field.html.twig

```diff
@@ -13,7 +13,6 @@
                       :helpText="grossHelpText"
                       :validation="validation"
                       :size="compact ? 'small' : 'default'"
-                      :min="0"
```

Comment:
Why not configureable?

Reply:
What kind of config do you expect? `allowNegative` flag?

Reply:
I expect the component is used also in the product / shipment section. It makes no sense to have there a negative number. To have the same functionality there I expect to make this configurable :)

Reply:
I changed this component BECAUSE it is used for pricing and I have negative prices. Certainly not a design decision I took myself but the API allows it and so should its UI.